test/extended/openstack/servergroup.go

```diff
@@ -6,80 +6,114 @@ import (
 
 	"github.com/gophercloud/gophercloud"
 	"github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/servergroups"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
 	g "github.com/onsi/ginkgo"
 	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	yaml "gopkg.in/yaml.v2"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
+	clientcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 )
 
 var _ = g.Describe("[sig-installer][Feature:openstack] The OpenStack platform", func() {
 	defer g.GinkgoRecover()
 
-	// OCP 4.5: https://issues.redhat.com/browse/OSASINFRA-1300
-	g.It("creates Control plane nodes in a server group", func() {
-		ctx := context.TODO()
+	var ctx context.Context
+	var computeClient *gophercloud.ServiceClient
+	var dc dynamic.Interface
+	var oc *exutil.CLI
+	var masterInstanceUUIDs []interface{}
+	var serverGroupName string
 
-		g.By("preparing a dynamic client")
-		cfg, err := e2e.LoadConfig()
-		o.Expect(err).NotTo(o.HaveOccurred())
-		dc, err := dynamic.NewForConfig(cfg)
-		o.Expect(err).NotTo(o.HaveOccurred())
+	oc = exutil.NewCLI("openstack")
 
-		g.By("getting the IDs of the Control plane instances and their Server group")
-		masterInstanceUUIDs := make([]interface{}, 0, 3)
-		var serverGroupName string
-		{
+	// OCP 4.5: https://issues.redhat.com/browse/OSASINFRA-1300
```

Comment:
This comment refers to the test "creates Control plane nodes in a server group"

Reply:
I'll move